### DISCUSSION TITLE
I want to sort my star list by drag and drop

### DISCUSSION THREAD

**Discussion by devcui** [community/community#56724] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/26159886/016eeda0-f3ac-4c91-9dd2-88bf066b4f06)

When I look at my list, the lists with similar names are very scattered, so I want to drag and sort it, can you introduce such a function?

**Comment by mimnifras** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> ![image](https://github.com/community/community/assets/26159886/016eeda0-f3ac-4c91-9dd2-88bf066b4f06)
> 
> When I look at my list, the lists with similar names are very scattered, so I want to drag and sort it, can you introduce such a function?
